### ISSUE TITLE
Consider adding the default pretrained model to the docker image

### ISSUE THREAD

**Issue by Remi-Gau** [DeepMReye/DeepMReye#36]
It will increase image size a bit but it may ensure better reproducibility by making sure that you can always use a given model for a given version of the docker image.

Any thoughts?

**Comment by CYHSM** [comment]
Yes, we could do that. Current filesize for the model weights is 417Mb, not sure how big the full docker image is?

**Comment by Remi-Gau** [comment]
Current size is 2.7 Go

https://hub.docker.com/layers/deepmreye/deepmreye/unstable/images/sha256-77cca96abd23d0ee09356732b750edc2072dd7dca92ea63dfd76264704a07481?context=explore

**Comment by Naubody** [comment]
@Remi-Gau, did you end up including the pretrained models in the docker image?

**Comment by Remi-Gau** [comment]
Is this the default model?

https://github.com/DeepMReye/DeepMReye/blob/9318b4f22f5c85b408036d48d5db6963b729960a/Dockerfile#L63

**Comment by Naubody** [comment]
This is the model trained on the most diverse data, so it would make sense to use it as the default model. Personally, however, I have mainly used dataset1to5.h5 in the past, which worked well for me. 

Other than my lack of direct experience using dataset1to6, I don't see a reason why not to use it by default, unless @CYHSM sees something I don't. Has anyone ever directly compared the performance of dataset1to6.h5 vs. dataset1to5.h5 for free viewing?

**Comment by CYHSM** [comment]
Sorry for the late reply, the dataset1to6 should be fine to use but as @Naubody said we have not quantified it yet and we have both used mainly the dataset1to5 so to be on the safe side I would use 1to5 as the default model
